bench/micro/path_bench.ml

```diff
@@ -8,56 +8,54 @@ let root = "."
-                           ; ( "reach long path from similar long path"
-                             , ( Filename.concat long_path "a"
-                               , Filename.concat long_path "b" ) )
-                           ; ( "reach short path from similar short path"
-                             , ( Filename.concat short_path "a"
-                               , Filename.concat short_path "b" ) )
-                           ]])
+let%bench_fun ("reach"
+                [@params
+                  t
+                  = [ "from root long path", (long_path, root)
+                    ; "from root short path", (short_path, root)
+                    ; "reach root from short path", (root, short_path)
+                    ; "reach root from long path", (root, long_path)
+                    ; ( "reach long path from similar long path"
+                      , (Filename.concat long_path "a", Filename.concat long_path "b") )
+                    ; ( "reach short path from similar short path"
+                      , (Filename.concat short_path "a", Filename.concat short_path "b") )
+                    ]])
   =
   let t, from = t in
   let t = Path.of_string t in
   let from = Path.of_string from in
   fun () -> ignore (Path.reach t ~from)
 ;;
 
-let%bench_fun ("Path.Local.relative" [@params
-                                       t
-                                       = [ "left root", (".", long_path)
-                                         ; "right root", (long_path, ".")
-                                         ; "short paths", (short_path, short_path)
-                                         ; "long paths", (long_path, long_path)
-                                         ]])
+let%bench_fun ("Path.Local.relative"
+                [@params
+                  t
+                  = [ "left root", (".", long_path)
+                    ; "right root", (long_path, ".")
+                    ; "short paths", (short_path, short_path)
+                    ; "long paths", (long_path, long_path)
+                    ]])
   =
   let x, y = t in
   let x = Path.Local.of_string x in
   fun () -> ignore (Path.Local.relative x y)
 ;;
 
-let%bench_fun ("Path.Local.append" [@params
-                                     t
-                                     = [ "left root", (".", long_path)
-                                       ; "right root", (long_path, ".")
-                                       ; "short paths", (short_path, short_path)
-                                       ; "long paths", (long_path, long_path)
-                                       ]])
+let%bench_fun ("Path.Local.append"
+                [@params
+                  t
+                  = [ "left root", (".", long_path)
+                    ; "right root", (long_path, ".")
+                    ; "short paths", (short_path, short_path)
+                    ; "long paths", (long_path, long_path)
+                    ]])
```

Comment:
Much more readable, only a bit surprising where the `[` starts. It is aligned with the content of the string? What if the start is `{foo|`, where is the indent? I think just 2 characters from the item to which the attribute is attached would be more consistent.

Reply:
It was in fact indented by 2 starting from the opening parenthese. Here's a more intuitive indentation: https://github.com/ocaml-ppx/ocamlformat/pull/2613
But because this is already changed, a better formatting is possible: https://github.com/ocaml-ppx/ocamlformat/pull/2615